sdk/core/azure-core/inc/azure/core/http/curl/curl connection_pool.hpp

```diff
@@ -27,6 +27,84 @@ namespace Azure { namespace Core { namespace Test {
 
 namespace Azure { namespace Core { namespace Http {
 
+  /**
```

Comment:
File name has a space in it, "curl connection_pool.hpp".  Please fix to have the underscore.  Ok to do so in a follow up PR if that makes the change easier to get in.